src/main/java/cloud/fogbow/rcs/core/intercomponent/RemoteFacade.java

```diff
@@ -25,6 +35,29 @@ public String requestService(String senderId, ServiceType serviceType) {
         return this.factory.makeCatalogService().requestService(senderId, serviceType);
     }
 
+    public List<ServiceType> getServices() {
```

Comment:
Migrate the implementation of this method to the **CatalogService** class and only make it called here.